apps/next-evans/next.config.js

```diff
@@ -6,27 +7,74 @@ const LANG = process.env.NEXT_PUBLIC_LANG || `en`;
 const nextConfig = {
   reactStrictMode: false,
   transpilePackages: [`@friends-library`, `x-syntax`],
+  experimental: {
```

Comment:
do we want this here? are we using the experimental app dir? or was this just a relic of some exploration?